File: testdata/patch/python/expected.py
def handle_resources_read(uri: str):
    resource = store.get(uri)
    if resource is None:
        return {"error": {"code": -32602, "message": "resource not found"}}
    return {"result": resource}


def read_resource(uri: str):
    if uri not in store:
        return {
            "error": {
                "code": -32602,
                "message": "not found",
            }
        }
    return {"result": store[uri]}

File: testdata/patch/js/expected.js
async function handleResourcesRead(uri) {
  const resource = store.get(uri);
  if (!resource) {
    return { error: { code: -32602, message: "resource not found" } };
  }
  return { result: resource };
}

// multi-line style
async function readResource(uri) {
  if (!store.has(uri)) {
    return {
      error: {
        code: -32602,
        message: "resource not found",
      },
    };
  }
  return { result: store.get(uri) };
}
